### DISCUSSION TITLE
Different Colors in App vs. Documentation

### DISCUSSION THREAD

**Discussion by MrNebel** [tailwindlabs/tailwindcss#11165] [Help] [answered]
Hello,

I'm currently writing a web app and I want the user to be able to choose an accent color for the pages.

After creating a badge component (editing Flowbite's pre-built), I noticed that my accent green is different from the green generated by TailwindCSS:

![image](https://user-images.githubusercontent.com/58656475/236555633-c9ed7692-99a5-4d28-9ceb-9ddc9aaceea8.png)

(Primary Badge is with the Accent Color, Green Badge with TailwindCSS Colors)
The Difference is Minor, but it's happening to all Colors.

Here is the HTML Code to create Both Badges:

```html
<span class="whitespace-nowrap font-medium px-2.5 py-0.5 rounded text-xs bg-primary-100 text-primary-800 dark:bg-primary-900/75 dark:text-primary-300">
    Primary Badge
</span>
<span class="whitespace-nowrap font-medium px-2.5 py-0.5 rounded text-xs bg-green-100 text-green-800 dark:bg-green-900/75 dark:text-green-300">
    Green Badge
</span>
```

This is my TailwindCSS configuration file:

```js
const colors = require("tailwindcss/colors");

module.exports = {
    darkMode: "class",
    content: [
        // ...
    ],
    theme: {
        fontFamily: {
            smooth: 'Nunito, system-ui, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif',
            sans: 'Montserrat, system-ui, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif',
        },
        extend: {
            animation: {
                "spin-slow": "spin 5s linear infinite",
            },
            colors: {
                primary: {
                    50: "rgb(var(--primary-50) / <alpha-value>)",
                    100: "rgb(var(--primary-100) / <alpha-value>)",
                    200: "rgb(var(--primary-200) / <alpha-value>)",
                    300: "rgb(var(--primary-300) / <alpha-value>)",
                    400: "rgb(var(--primary-400) / <alpha-value>)",
                    500: "rgb(var(--primary-500) / <alpha-value>)",
                    600: "rgb(var(--primary-600) / <alpha-value>)",
                    700: "rgb(var(--primary-700) / <alpha-value>)",
                    800: "rgb(var(--primary-800) / <alpha-value>)",
                    900: "rgb(var(--primary-900) / <alpha-value>)",
                    950: "rgb(var(--primary-950) / <alpha-value>)",
                },
            },
        },
        colors: {
            transparent: "transparent",
            zinc: colors.zinc,
            gray: colors.gray,
            red: colors.red,
            orange: colors.orange,
            yellow: colors.yellow,
            green: colors.green,
            teal: colors.teal,
            sky: colors.sky,
            indigo: colors.indigo,
            purple: colors.purple,
            pink: colors.pink,
            black: colors.black,
            white: colors.white,
        },
    },

    plugins: [
        require("postcss-import"),
        require("tailwindcss"),
        require("autoprefixer"),
        require("@tailwindcss/forms"),
        require("flowbite/plugin"),
    ],
};
```

Here is a Snippet of my SCSS File for generating the CSS Variables:
```scss
body[data-themecolor="green"] *,
    body[data-themecolor="green"] ::before,
    body[data-themecolor="green"] ::after {
        --primary-50: 240 252 244;
        --primary-100: 220 252 231;
        --primary-200: 187 247 208;
        --primary-300: 134 239 172;
        --primary-400: 74 222 128;
        --primary-500: 34 197 94;
        --primary-600: 22 163 74;
        --primary-700: 21 128 61;
        --primary-800: 22 101 52;
        --primary-900: 20 83 45;
        --primary-950: 5 46 22;
    }
```

If I Inspect the Element in the Dev Tools I get for the Green:
```css
.bg-green-100 {
    --tw-bg-opacity: 1;
    background-color: rgb(222 247 236 / var(--tw-bg-opacity));
}
```

`222, 247, 236 = #def7ec` (Changed to HEX via VSCode. --tw-bg-opacity is set to 1)

The Docs show that the Color for green-100 is `#dcfce7`.

`#def7ec` != `#dcfce7`

The `#dcfce7` is also set into the `src/public/colors.js` file, where all Colors are Stored.

So after all this Preparation.. 

What is happening here??

Edit: I've checked this post [#3861](https://github.com/tailwindlabs/tailwindcss/discussions/3861) and made sure that @tailwindcss/ui is not installed on my System or required by any other Library

Edit: I've also Implemented Dark Mode, where the Difference is clearly more visible than in Light mode:
![image](https://user-images.githubusercontent.com/58656475/236560368-bae47424-f571-46f0-a6fb-5d36c0fe1f01.png)

Thanks in advance. Maybe someone can help me to understand this phenomenon

**Comment by wongjn** [comment, marked as answer]
`#dcfce7` is the color for `green.100` in Tailwind by default, but Flowbite overrides this to be `#def7ec`[^1][^2].
Similarly, for dark mode, `--primary-900`'s `20 83 45` ≠ `1 71 55` (Flowbite's `green.900`[^2][^3])

[^1]: https://github.com/themesberg/flowbite/blob/0181f3b1b74944b1bb8dce87c3955c1c5df75c6a/plugin.js#LL514C32-L514C38
[^2]: https://flowbite.com/docs/customize/colors/
[^3]: https://github.com/themesberg/flowbite/blob/0181f3b1b74944b1bb8dce87c3955c1c5df75c6a/plugin.js#LL522C32-L522C38
